tests/phpunit/framework/class-llms-settings-page-test-case.php

```diff
@@ -98,7 +100,7 @@ public function test_get_settings() {
 
 		$mock   = array_keys( $settings );
 		$actual = $this->get_settings_ids();
-		$this->assertEquals( $mock, $actual );
+		$this->assertEqualSets( $mock, $actual );
```

Comment:
WHat test is failing as a result of settings order?

The order kind of does matter here as the settings *should not* be being displayed in a random order on the frontend so they also should not be returned in a random order. The order is hardcoded into the arrays.

Reply:
alright, going to fix this.